hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmVolumeArgs.java

```diff
@@ -84,6 +97,14 @@ private OmVolumeArgs(String adminName, String ownerName, String volume,
     this.updateID = updateID;
   }
 
+  public long getRefCount() {
+    assert (refCount >= 0);
+    return refCount;
+  }
+
+  public void setRefCount(long refCount) {
```

Comment:
Good idea.

Reply:
Is there any reason to leave the setRefCount here? Still seems error prone, lots of people don't read javadocs before calling things : ) If we have a use case now we can leave it but this doesn't seem like a good thing to put in "just in case".